### ISSUE TITLE
Social buttons are not accessible

### ISSUE THREAD

**Issue by tonifisler** [swiss/styleguide#222]
![capture d ecran 2014-08-08 a 17 07 58](https://cloud.githubusercontent.com/assets/1708450/3858493/5b3ebb6a-1f0e-11e4-8859-2989337b3f83.png)

As we are using a plugin and everything is handled by it, we cannot change this behavior.
